src/NuGetPackageVerifier/Program.cs

```diff
@@ -25,6 +25,13 @@ public static int Main(string[] args)
 
             // TODO: Get this from the command line
             var ignoreAssistanceMode = IgnoreAssistanceMode.ShowAll;
+            var hideInfoLogs = false;
+
+            if (args.Length > 0 && string.Equals("--quiet", args[0], StringComparison.OrdinalIgnoreCase))
```

Comment:
Ah I see. This works for now. But I think it would be nice to have the SUMMARY and Total time taken not hidden. How about we have them as info and make the rest as Trace or Verbose? And --quiet will hide the Trace/Verbose messages